opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/MockTracingConfiguration.java

```diff
@@ -31,6 +30,7 @@
  */
 @Configuration
 @EnableAutoConfiguration
+@PropertySource(value = {"classpath:test.properties"})
```

Comment:
@geoand @gytis is there a way how configuration can add a single property? Only this worked to me.

Reply:
@pavolloffay If I correctly understand what you are trying to do, then [this](https://github.com/snowdrop/opentracing-tracer/blob/master/opentracing-tracer-jaeger-starter/src/test/java/me/snowdrop/opentracing/tracer/jaeger/starter/JaegerTracerCreatedSpringTest.java) should be what you are looking for

Reply:
It does not work here other tests annotated with `@SpringBootTest` defines this configuration in `classes` attribute. When I add that annotation directly on this class nothing happens.

Reply:
I'm not sure what you're trying to do. Could you explain?

Reply:
`MockTracingConfiguration` is a configuration class which is added to classes attribute in many tests annotated with `@SpringBootTest(classes=,...)`. 

I want that `MockTracingConfiguration` change a configuration property - `opentracing.spring.web.skipPattern=/notTraced` (defined in `WebTracingProperties`) - so the web tracing integration will not trace `/notTraced` path.

Reply:
I could maybe define a beanPostProcessing on `WebTracingProperties` but maybe there is a more convenient solution

Reply:
It seems to me that the property should be set by the Spring tests, not on the Spring configuration class